components/edit-collective/EditCollectiveForm.js

```diff
@@ -372,7 +372,7 @@ class EditCollectiveForm extends React.Component {
       return (
         <EditTiers
           title="Tiers"
-          types={['TIER', 'MEMBERSHIP', 'SERVICE', 'PRODUCT', 'DONATION']}
+          types={['TIER', 'MEMBERSHIP', 'SERVICE', 'PRODUCT', 'DONATION', 'TICKET']}
```

Comment:
If we're doing that, we're adding tickets to all Collectives, that's wrong. I have some specs to avoid that.